root/user/UserProfile.js

```diff
@@ -277,6 +280,13 @@ const UserProfileInformation = ({
           {memberSince}
         </UserProfileProperty>
 
+        {/* Only relevant to logged in users, avoid more public shame */}
+        {viewingUser && restrictions.length ? (
+          <UserProfileProperty name={l(addColonText('Active restrictions'))}>
```

Comment:
Suggestion by @Aerozol in an actionnable way:
```suggestion
          <UserProfileProperty name={l(addColonText('Restrictions'))}>
```